msteams-platform/samples/virtual-tables-api.md

```diff
@@ -0,0 +1,229 @@
+
+  `<cookie pagenumber=3 />`
+
+# [Request](#tab/request1)
+
+```http
+     GET [Organization URL]/api/data/v9.2/m365_graphevents?$top=2&$skiptoken=<cookie pagenumber=3> 
+```
+
+# [Response](#tab/response1)
+
+```json
+
+{
+  "@odata.context": "[Organization URI]/api/data/v9.0/$metadata#m365_graphdevents", 
+  "value": [
+   { 
+      "@odata.etag": "W/\"{FA93AF7C-1F45-4714-85A5-BB95EB86E1E5}\"", 
+      "m365_graphdeventid": "3d59a7e2-ec83-d0b3-270e-8ad676622027", 
+      "m365_subject": "Important meeting", 
+      …
+    }, 
+    {
+      "@odata.etag": "W/\"{3938D549-1AEF-46A5-BF3C-38472AD934C2}\"", 
+      "m365_graphdeventid": "f50aae23-6644-3d35-66d7-e3c5a979dad3", 
+      "m365_subject": "Another important meeting", 
+      …
+    } 
+  ] 
+}
+
+```
+
+---
+
+> [!Note]
+> The response will not include the `@nextLink` property. If your use case requires the next page link to be returned, you can use the odata.maxpagesize preference header described in section 1 instead of passing the $top URI parameter.
+
+Currently the following virtual tables support fetching a specific page:
+
+* Graph Booking Appointment
+* Graph Calendar Event
+* Graph Chat Message
+
+You can pass a fetch XML as an XML-encoded string. With the fetch XML option, you can specify several query preferences. The pagination specific options are page (page number) and count (page size). The following XML specifies the page number and size:
+
+  `<fetch version="1.0" mapping="logical" returntotalrecordcount="true" page="<Page Number>" count="<Page Size>"></fetch>`
+
+# [Request](#tab/request2)
+
+```http
+GET [Organization URL]/api/data/v9.2/m365_graphevents?$fetchXml=<fetch version="1.0" mapping="logical" returntotalrecordcount="true" page="3" count="2"></fetch> 
+
+```
+
+# [Response](#tab/response2)
+
+```json
+{ 
+
+    "@odata.context": "[Organization URI]/api/data/v9.0/$metadata#m365_graphdevents", 
+    "@Microsoft.Dynamics.CRM.fetchxmlpagingcookie": "<cookie pagenumber=\"3\" pagingcookie=\"\" istracking=\"False\" />", 
+    "value": [ 
+        { 
+            "@odata.etag": "W/\"{FA93AF7C-1F45-4714-85A5-BB95EB86E1E5}\"", 
+            "m365_graphdeventid": "3d59a7e2-ec83-d0b3-270e-8ad676622027", 
+            "m365_subject": "Important meeting", 
+            …
+        }, 
+        { 
+            "@odata.etag": "W/\"{3938D549-1AEF-46A5-BF3C-38472AD934C2}\"", 
+            "m365_graphdeventid": "f50aae23-6644-3d35-66d7-e3c5a979dad3", 
+            "m365_subject": "Another important meeting", 
+            …
+        } 
+    ] 
+} 
+
+```
+
+---
+
+The following virtual tables support the count property to be passed as part of the fetchXml option:
+
+* Graph Chat
+* Graph Chat Message
```

Comment:
Graph Chat and Graph Chat Message table won't be supported, so these items can be removed.